src/script.js

```diff
@@ -73,15 +75,12 @@ $j(document).ready(() => {
 		.prop('checked', true)
 		.trigger('click');
 
-	// Refresh the buttonset UI so that newly checked location is displayed
-	$j('.typeRadio').buttonset('refresh');
-
 	// Disable initial game setup until browser tab has focus
 	window.addEventListener('blur', G.onBlur.bind(G), false);
 	window.addEventListener('focus', G.onFocus.bind(G), false);
 
 	// Focus the form to enable "press enter to start the game" functionality
-	$j('#p2').focus();
+	$j('input[type="radio"]').focus();
```

Comment:
This causes the focus on `Dragon Bones`, requiring a `Tab` press in order to be able to just start the game using `Spacebar`.

Reply:
Spacebar should never submit a form... unless the submit button is focused... I apparently typo'd when I searched for p2 in the document which is why I changed it, I'll change that one back.